src/index.jsx

```diff
@@ -22,30 +21,18 @@ import './index.scss';
 import App from './components/App';
 import ErrorPage from './common/components/ErrorPage';
 
-if (configuration.ENVIRONMENT !== 'production') {
-  import(/* webpackChunkName: "react-axe" */ 'react-axe')
-    .then(({ default: axe }) => axe(React, ReactDOM, 1000));
-}
-
-const apiClient = getAuthenticatedAPIClient({
-  appBaseUrl: configuration.BASE_URL,
-  authBaseUrl: configuration.LMS_BASE_URL,
-  loginUrl: configuration.LOGIN_URL,
-  logoutUrl: configuration.LOGOUT_URL,
-  csrfTokenApiPath: configuration.CSRF_TOKEN_API_PATH,
-  refreshAccessTokenEndpoint: configuration.REFRESH_ACCESS_TOKEN_ENDPOINT,
-  accessTokenCookieName: configuration.ACCESS_TOKEN_COOKIE_NAME,
-  userInfoCookieName: configuration.USER_INFO_COOKIE_NAME,
-  csrfCookieName: configuration.CSRF_COOKIE_NAME,
-  loggingService: NewRelicLoggingService,
-});
+let apiClient = null;
```

Comment:
We need the apiClient all over in this file, so it needs to be scoped to the top.  (i.e., it can't be in a try/catch)